webui/Caddyfile

```diff
@@ -18,7 +18,7 @@ header / {
   # Disallow the site to be rendered within a frame (clickjacking protection)
   X-Frame-Options "sameorigin"
   # CSP
-  Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline'; img-src 'self' data: www.gravatar.com; style-src 'self' 'unsafe-inline';  font-src 'self' data: ;"
+  Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval'; img-src 'self' data: www.gravatar.com; style-src 'self' 'unsafe-inline';  font-src 'self' data: ;"
```

Comment:
Can the `unsafe-eval` be dropped now that the script tag is not being emitted by elm?

Reply:
I confirmed this can be removed now.